packages/ember-engines/package.json

```diff
@@ -103,22 +102,17 @@
     "calculate-cache-key-for-tree": "^2.0.0",
     "ember-asset-loader": "^1.0.0",
     "ember-cli-babel": "^7.18.0",
+    "ember-cli-htmlbars": "^5.3.2",
     "ember-cli-preprocess-registry": "^3.3.0",
     "ember-cli-string-utils": "^1.1.0",
     "ember-cli-version-checker": "^5.1.2",
     "lodash": "^4.17.11"
   },
   "peerDependencies": {
-    "ember-source": "^3.24.1 || 4 || 5",
-    "@ember/legacy-built-in-components": "*"
-  },
-  "peerDependenciesMeta": {
-    "@ember/legacy-built-in-components": {
-      "optional": true
-    }
+    "ember-source": "^3.24.1 || 4 || >=5"
   },
   "engines": {
-    "node": "14.* || 16.* || >= 18"
+    "node": "16.* || >= 18"
```

Comment:
@void-mAlex I think we should do this in separate PR and probably go 18+, wdyt?

Reply:
happy to do 18+, but not sure about value of doing it as a separate pr, they still need to go in together
happy to break them up but more work imo when you really end up doing two separate breaking changes

Reply:
I agree with @void-mAlex here, we don't need a separated PR when we can hit two birds with one stone ; )